pkg/sql/testdata/pgoidtype

```diff
@@ -162,7 +163,34 @@ CREATE DATABASE otherdb
 statement ok
 SET DATABASE = otherdb
 
+## a non-root user can't get the OID of a table from a different database
+
 user testuser
 
-query error relation 'a' does not exist
+query error table "a" does not exist
 SELECT 'a'::regclass
+
+user root
+
+statement ok
+CREATE TABLE a (id INT PRIMARY KEY, foo STRING)
+
+## There is now a table named 'a' in both the database 'otherdb' and the
+## database 'test'. The following query shows that the root user can still
+## determine the OID of the table 'a' by using a regclass cast, despite the
+## fact that the root user has visibility into both of the tables. The 'a' that
+## gets selected should be the 'a' that exists in the current database.
```

Comment:
But that's not what the code does, right? I think the sentence is "the 'a' that gets selected is the first 'a' that can be found in either the current database or (next) any database in the search path".